pxf/settings.gradle

```diff
@@ -19,9 +19,10 @@
 
 rootProject.name = 'pxf'
 
+include 'pxf'
```

Comment:
settings.gradle must have every project included. Without which we would get the following error: Project with path 'pxf-service' could not be found in root project 'pxf'.